### PR TITLE
Bug fix for metrics and update to config metadata loader

### PR DESCRIPTION
**Pull request by svidhani**
Metrics bugfix - stack had the older code which was taken before the merge with v.1.8.1. 
Update to metrics mustache values did not work and config metadata loader is updated to handle that.
Updated create-serverless-service swagger file
Update build pack api missing a closing brace

### CODE REVIEW

File: builds/jazz-build-module/service-configuration-data-loader.groovy [hunk 1]
```diff
@@ -24,14 +24,15 @@ echo "Service configuration module loaded successfully"
...
+
     config_loader = config
     setRoleARN(role_arn)
     setRegion(region)
     setRoleId(role_id)
     setJenkinsUrl(jenkins_url)
     setCurrentEnvironment(current_environment)
     setServiceName(service_name)
-    setUtilModule(utilModule)
+    setUtilModule(utilModule
```
Comment (devsatishm) [inline builds/jazz-build-module/service-configuration-data-loader.groovy]: Missing )

Comment (svidhani) [reply builds/jazz-build-module/service-configuration-data-loader.groovy]: fixed

File: builds/jenkins-build-pack-api/Jenkinsfile [hunk 2]
```diff
@@ -302,32 +302,33 @@ node() {
...
-					}
+					  }
 
-					if (domain && domain == "jazz") {
-						serviceConfigdata.setLogStreamPermission(config)
-					}
+            if (domain && domain == "jazz") {
+              serviceConfigdata.setLogStreamPermission(config)
+            }
+          }
```
Comment (svidhani) [inline builds/jenkins-build-pack-api/Jenkinsfile]: this is the only change

File: core/jazz_create-serverless-service/swagger/swagger.json [hunk 3]
```diff
@@ -282,14 +282,13 @@
...
                 },
-				"require_internal_access": {
+				        "require_internal_access": {
                     "type": "boolean",
                     "description":"Does service require access to internal SLF network and resources"
                 },
                 "deployment_targets": {
                     "type": "object",
                     "description": "Deployment targets list for the service",
-                    "required": false,
```
Comment (devsatishm) [inline core/jazz_create-serverless-service/swagger/swagger.json]: Why did we remove this?

Comment (svidhani) [reply core/jazz_create-serverless-service/swagger/swagger.json]: that is not a valid swagger.

Comment (devsatishm) [reply core/jazz_create-serverless-service/swagger/swagger.json]: You will still need to add this as required in some part of swagger. `deployment_targets` is a required field.

Comment (svidhani) [reply core/jazz_create-serverless-service/swagger/swagger.json]: updated

**Pull request merged**